### ISSUE TITLE
Autolayout for nodes and edges?

### ISSUE THREAD

**Issue by twist900** [emumba-com/react-js-diagrams#1]
Any plans on adding autolayout functionality for nodes and edges?

**Comment by umarashfaq** [comment]
Can you please explain bit more what you mean by autolayout? Is it re-arrangement of nodes and edges so they are aligned?

**Comment by twist900** [comment]
Yes, is that on the list? I see that it is a fork, but it looks like it is being actively maintained.

**Comment by umarashfaq** [comment]
Yes autolayout would be a very useful functionality to have here. I think it'd be also be helpful in the project for which I forked this repo. 

However, in my experience working with graph rendering, determining nodes position automatically is a challenge. Let me know if you are aware of any algorithm that's applicable here. It'd also be helpful if you know any application/framework/site to draw inspiration from.
